app/frontend/src/i18n/config.ts

```diff
@@ -24,6 +25,10 @@ export const supportedLngs: { [key: string]: { name: string; locale: string } }
     ja: {
         name: "日本語",
         locale: "ja-JP"
+    },
+    ptBR: {
+        name: "Português",
+        locale: "pt-BR"
```

Comment:
Thank you Glaucia! If someone comes in with pt-PT as the language code, would they just get English translations then? Is that how it's normally done? I remember learning that the pronunciation of pt-PT is very different from pt-BR in my portuguese class, but I don't know how different the words are, so I always wondered how that was handled for i18n.

Reply:
> Thank you Glaucia! If someone comes in with pt-PT as the language code, would they just get English translations then? Is that how it's normally done? I remember learning that the pronunciation of pt-PT is very different from pt-BR in my portuguese class, but I don't know how different the words are, so I always wondered how that was handled for i18n.

Yes, there are many differences. But I've heard that our Portuguese cousins prefer to learn Brazilian Portuguese. But if it comes up, let me know and I'll help you out

Reply:
Let me include here:

```json
 name: "Português Brasileiro",
```

Reply:
Done! I made the change